app/client/src/redux/ducks/authDuck.js

```diff
@@ -88,6 +87,29 @@ export const register = (name, email, password) => async (dispatch) => {
   }
 };
 
+export const logout = () => async (dispatch) => {
```

Comment:
Logout needs to clear the access token from redux memory

Reply:
The logout action dispatches LOGOUT which clears the auth state